config/webpack.config.js

```diff
@@ -1,10 +1,13 @@
 const webpack = require('webpack');
 const path = require('path');
 const fs = require('fs');
-const dependencies = require(path.join(
+
+const { dependencies } = require(path.join(
     process.cwd(),
-    'package.json',
-)).dependencies;
+    'node_modules',
+    'pc-nrfconnect-shared',
```

Comment:
Is this really correct? I know that there were some changes to this on the master branch and I fear this might be a merge gone wrong.

Reply:
This is correct, though it wasn't my addition. Chun added it when we had some problems getting the apps to build with the new config, a file resolution error somewhere. Somehow this solves that issue.

Reply:
Chun did change this in https://github.com/NordicSemiconductor/pc-nrfconnect-shared/commit/d0058090606ff8559731fefa83d8f340106ceba3#diff-a58d55bdb5770c78ad512f8e91f8d051 but he changed it again in https://github.com/NordicSemiconductor/pc-nrfconnect-shared/commit/8b77d7afc373f8f185e450ea83fa3e7aeea22563#diff-a58d55bdb5770c78ad512f8e91f8d051 because we had problems with installing local apps. So the current version in `master` looks like this and yours is different, so please check this again: https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/184a1054e30b5464a8b070e7ada02100b4a9a33b/config/webpack.config.js#L4-L7